service/policy/db/query.sql

```diff
@@ -87,13 +87,15 @@ OFFSET @offset_;
 
 -- name: GetKeyAccessServer :one
 SELECT 
-    id,
-    uri, 
-    public_key, 
-    name,
+    kas.id,
+    kas.uri, 
+    kas.public_key, 
+    kas.name,
     JSON_STRIP_NULLS(JSON_BUILD_OBJECT('labels', metadata -> 'labels', 'created_at', created_at, 'updated_at', updated_at)) AS metadata
-FROM key_access_servers
-WHERE id = $1;
+FROM key_access_servers AS kas
+WHERE (sqlc.narg('id')::uuid IS NULL OR kas.id = sqlc.narg('id')::uuid)
```

Comment:
Why not use the `@` macro here instead? I think we are using it elsewhere already

```suggestion
WHERE (@id::uuid IS NULL OR kas.id = @id::uuid)
```

Reply:
That is a shortcut for `sqlc.arg()` not `sqlc.narg()` which are nullable parameters. Allowing nullable parameters allows us to query by different parameters.